zebra/connected.c

```diff
@@ -193,6 +193,7 @@ void connected_up(struct interface *ifp, struct connected *ifc)
 	uint32_t count = 0;
 	struct connected *c;
 	bool install_local = true;
+	bool p_same;
```

Comment:
I don't have a problem with a /32 local route and a /32 connected route.  The connected one is currently preferred, although I believe the local one should be.  What problem are you trying to solve?